### PR TITLE
repo: Release 1.25.0

### PR DESCRIPTION
**Pull request by KBaichoo**
Commit Message:
Additional Description: Aiming to submit early tomorrow
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Comment by KBaichoo** [comment]
/assign @phlax 

cc @alyssawilk @mattklein123

File: changelogs/1.23.3.yaml [hunk 1]
```diff
@@ -1,2 +1 @@
 date: December 2, 2022
-
```
Comment (KBaichoo) [inline changelogs/1.23.3.yaml]: cc @phlax 
Seems like the automated tool generated a file with  empty line which conflicted with format checks

Comment (phlax) [reply changelogs/1.23.3.yaml]: its possible - we added the linter fairly recently - i reckon we can just fix the file for now - ill raise a ticket ...

https://github.com/envoyproxy/envoy/issues/24918

**Comment by KBaichoo** [comment]
prefer #24970

**Pull request closed without merging**